Return-Path: dmarlin@redhat.com
Received: from zmta01.collab.prod.int.phx2.redhat.com (LHLO
 zmta01.collab.prod.int.phx2.redhat.com) (10.5.5.31) by
 zmail14.collab.prod.int.phx2.redhat.com with LMTP; Fri, 2 Mar 2012 13:20:22
 -0500 (EST)
Received: from localhost (localhost.localdomain [127.0.0.1])
	by zmta01.collab.prod.int.phx2.redhat.com (Postfix) with ESMTP id 472DB114015;
	Fri,  2 Mar 2012 13:20:22 -0500 (EST)
Received: from zmta01.collab.prod.int.phx2.redhat.com ([127.0.0.1])
	by localhost (zmta01.collab.prod.int.phx2.redhat.com [127.0.0.1]) (amavisd-new, port 10024)
	with ESMTP id 6bwpcty13jPD; Fri,  2 Mar 2012 13:20:22 -0500 (EST)
Received: from int-mx10.intmail.prod.int.phx2.redhat.com (int-mx10.intmail.prod.int.phx2.redhat.com [10.5.11.23])
	by zmta01.collab.prod.int.phx2.redhat.com (Postfix) with ESMTP id 335C6114003;
	Fri,  2 Mar 2012 13:20:22 -0500 (EST)
Received: from fedora15-1.farm.hsv.redhat.com (fedora15-1.farm.hsv.redhat.com [10.15.4.181])
	by int-mx10.intmail.prod.int.phx2.redhat.com (8.14.4/8.14.4) with ESMTP id q22IKLKB027218;
	Fri, 2 Mar 2012 13:20:21 -0500
From: "d.marlin" <dmarlin@redhat.com>
To: pjones@redhat.com
Cc: "d.marlin" <dmarlin@redhat.com>
Subject: [PATCH] Add support to new-kernel-pkg to recognize ARCH == armv5tel needed for Kirkwood based systems.
Date: Fri,  2 Mar 2012 12:19:29 -0600
Message-Id: <1330712369-15567-1-git-send-email-dmarlin@redhat.com>
X-Scanned-By: MIMEDefang 2.68 on 10.5.11.23

From: "d.marlin" <dmarlin@redhat.com>

Signed-off-by: David A. Marlin <dmarlin@redhat.com>
---
 new-kernel-pkg |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/new-kernel-pkg b/new-kernel-pkg
index 6b94d2f..d3778d1 100755
--- a/new-kernel-pkg
+++ b/new-kernel-pkg
@@ -78,7 +78,7 @@ elif [ $ARCH = 's390' -o $ARCH = 's390x' ]; then
     lilo=/sbin/zipl
     runLilo="yes"
     isx86=""
-elif [ $ARCH = 'armv7l' -o $ARCH = 'armv7hl' ]; then
+elif [ $ARCH = 'armv7hl' -o $ARCH = 'armv7l' -o $ARCH = 'armv5tel' ]; then
     machine=$(grep "^Hardware" /proc/cpuinfo | sed 's/Hardware\s*:\s*//')
     liloConfig=""
     bootPrefix=/boot
-- 
1.7.6.5

